### PR TITLE
Refactor `peer_message.go`

### PR DESCRIPTION
**Pull request by muXxer**
Cluster tests passed for me.

### CODE REVIEW

File: packages/peering/lpp/lppPeer.go [hunk 1]
```diff
@@ -201,6 +201,8 @@ func (p *peer) IsAlive() bool {
 }
 
 // Await implements peering.PeerSender interface for the remote peers.
+//
+//nolint:revive
```
Comment (Juliusan) [inline packages/peering/lpp/lppPeer.go]: Github notifies:
```directive `//nolint:revive` is unused for linter "revive" (nolintlint)```
Maybe it should be removed.

File: packages/peering/lpp/lppNetImpl.go [hunk 2]
```diff
@@ -426,6 +426,8 @@ func (n *netImpl) NumUsers() int {
 }
 
 // Await implements peering.PeerSender for the Self() node.
+//
+//nolint:revive
```
Comment (Juliusan) [inline packages/peering/lpp/lppNetImpl.go]: Github notifies:
directive `//nolint:revive` is unused for linter "revive" (nolintlint)
Maybe it should be removed.

**Review by Juliusan — APPROVED** [review]

File: packages/peering/lpp/lppPeer.go (hunk 1 above)
Comment (muXxer) [reply packages/peering/lpp/lppPeer.go]: That will get a problem with the next golangci version, but there are tons of other errors we need to address in another PR anyway. I will remove it for now :+1:

File: packages/peering/lpp/lppNetImpl.go (hunk 2 above)
Comment (muXxer) [reply packages/peering/lpp/lppNetImpl.go]: That will get a problem with the next golangci version, but there are tons of other errors we need to address in another PR anyway. I will remove it for now :+1:

**Pull request merged**